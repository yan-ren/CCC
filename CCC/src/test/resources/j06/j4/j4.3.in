7
2
4
5
0
0
