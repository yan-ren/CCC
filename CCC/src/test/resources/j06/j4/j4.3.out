Cannot complete these tasks.  Going to bed.
